project/ProjectDependencies.scala

```diff
@@ -22,6 +22,8 @@ object ProjectDependencyVersions {
   val Log4jVersion = "2.5"
   val ProtobufVersion = "2.5.0"
   val SparkVersion = "1.6.1"
+  val VertxVersion = "3.0.0"
+  val ExampleVertxVersion = "3.3.2"
```

Comment:
The adapter should use Vert.x 3.0.0 as a provided dependency to not force the user into a specific Vert.x version because of breaking changes between minor versions (see https://github.com/vert-x3/wiki/wiki/3.1.0---Breaking-changes).